src/migrations/201602231400_initial.js

```diff
@@ -0,0 +1,107 @@
+'use strict'
+
+function createAccountsTable (knex) {
+  return knex.schema.createTableIfNotExists('accounts', table => {
+    table.increments()
+    table.string('name').unique()
+    table.decimal('balance', 10, 2)
+    table.string('connector', 1024)
+    table.string('password_hash')
+    table.text('public_key')
+    table.boolean('is_admin')
+    table.boolean('is_disabled')
+    table.string('fingerprint').index('fingerprint')
+  })
+}
+
+function createFulfillmentsTable (knex) {
+  return knex.schema.createTableIfNotExists('fulfillments', table => {
+    table.increments()
+    table.uuid('transfer_id').unique().index()
+    table.json('condition_fulfillment')
+  })
+}
+
+function createEntryGroupsTable (knex) {
+  return knex.schema.createTableIfNotExists('entry_groups', table => {
+    table.increments()
+    table.datetime('created_at')
+    table.datetime('updated_at')
+  })
+}
+
+function createEntriesTable (knex) {
+  return knex.schema.createTableIfNotExists('entries', table => {
+    table.increments()
+    table.integer('entry_group')
+    table.uuid('transfer_id')
+    table.integer('account')
+    table.decimal('balance', 10, 2)
+    table.datetime('created_at')
+    table.datetime('updated_at')
+  })
+}
+
+function createNotificationsTable (knex) {
+  return knex.schema.createTableIfNotExists('notifications', table => {
+    table.uuid('id').primary()
+    table.uuid('subscription_id')
+    table.uuid('transfer_id')
+    table.integer('retry_count')
+    table.datetime('retry_at').index()
+    table.index(['subscription_id', 'transfer_id'], 'subscription_transfer')
+  })
+}
+
+function createSubscriptionsTable (knex) {
+  return knex.schema.createTableIfNotExists('subscriptions', table => {
+    table.uuid('id').primary()
+    table.string('owner', 1024)
```

Comment:
Why use 1024 as the length? I noticed, in other tables, some `string` datatypes don't have a length specified.